provider/resources.go

```diff
@@ -16,22 +16,29 @@ package hcloud
 
 import (
 	"fmt"
+	_ "embed"
```

Comment:
The required comment explaining blank imports is not being added from https://github.com/pulumi/upgrade-provider/blob/aqiu/code-migration-ast/migrations/migrations.go#L29-L33

Reply:
I don't think we actually need this comment. We can assume that go programmers understand what `_ "${PKG}"` means.

Reply:
The lint test will complain and fail though, unless we're okay with ignoring that

Reply:
Ok. Then we need to keep it.